Author: Agustin Martin Domingo <agmartin@debian.org>
Description: Work around `window-dedicated-p' different syntax for XEmacs

XEmacs `window-dedicated-p' requires window name, it is not optional.

Xemacs-only:
Reverts upstream:

* Dmitry Antipov (Mon Aug 5 18:26:57 2013 +0400)
    
        textmodes/ispell.el (ispell-command-loop): Do not call to
        `selected-window' where it is assumed by default.

--- a/support/emacsen/ispell.el
+++ b/support/emacsen/ispell.el
@@ -2233,7 +2233,7 @@
 	(window-min-height (min window-min-height
 				ispell-choices-win-default-height))
 	(command-characters '( ?  ?i ?a ?A ?r ?R ?? ?x ?X ?q ?l ?u ?m ))
-	(dedicated (window-dedicated-p))
+	(dedicated (window-dedicated-p (selected-window)))
 	(skipped 0)
 	char num result textwin dedicated-win)
 
